Azure/azure-ad/ingest/parser.yml

```
@@ -0,0 +1,86 @@
+name: azure-ad
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+  - name: azuread
+  - name: azuread-filters
+  - name: set_action_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+  - name: set_if_message_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+
+stages:
+  azuread:
+    actions:
+      - set:
+          service.type: "ldap"
+          service.name: "active directory"
+          log.hostname: "azureactivedirectory"
+          host.hostname: "azureactivedirectory"
+          action.name: '{{parsed_event.message.operationName}}'
+          user.id: '{{ parsed_event.message.userId}}'
+          user.name: '{{ parsed_event.message.userPrincipalName}}'
+          action.outcome: '{{parsed_event.message.properties.result}}'
+          source.ip: '{{parsed_event.message.ipAddress}}'
+          user_agent.original: '{{parsed_event.message.userAgent}}'
+          action.type: "{{parsed_event.message.properties.operationType |lower}}"
+
+      - set:
+          action.target: "user"
+        filter: "{{parsed_event.message.userId != null}}"
+
+  azuread-filters:
+    actions:
+      - set:
+          action.name: "authentication"
+          event.category: "authentication"
+          event.type: "start"
```

My bad. This field must be an array.
```suggestion
          event.type: ["start"]
```

---

Applied in https://github.com/SEKOIA-IO/intake-formats/pull/53/commits/1d1670493cf319f21c9a69fd4b2a32938bcc6b83